docs/src/rules/prefer-destructuring.md

```diff
@@ -73,138 +81,112 @@ let bar;
 
 :::
 
-Examples of **incorrect** code when `enforceForRenamedProperties` is enabled:
+Alternatively, you can use separate configurations for different assignment types. It accepts 2 other keys instead of `array` and `object`.
```

Comment:
```suggestion
Alternatively, you can use separate configurations for different assignment types. The first argument accepts two other keys instead of `array` and `object`.
```